forms-flow-web/src/components/ServiceFlow/list/sort/NoFilterSelected.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../../ServiceFlow.scss';
+import NoFilterSelectedImage from './empty design.svg';
+
+function noFilterSelected() {
+    return (
+        <div>
+            <img src={NoFilterSelectedImage}  style={{ width : "100%" , height : "300PX"  }}/>
+            <h1 style={{ textAlign: 'center', lineHeight: '1.5' }}>Select a task in the List</h1>
```

Comment:
text should come with internationalistaion .

Reply:
I will do internationalization as a separate PR.